### PR TITLE
Docs Tutorial

### PR DESCRIPTION
**Pull request by coffee-cup**
Tutorial that creates this https://github.com/prodo-ai/github-prs.

There are probably lotst fo speling mistaes an' grammer mistakes. Happy to pair with someone to go over any larger changes about content. I tried to strike a balance between no assumed knowledge and conciseness, but that is difficult to do.

image

### CODE REVIEW

File: docs/content/05_tutorials/github-prs.md [hunk 1]
```diff
@@ -0,0 +1,493 @@
...
+The app created with create-prodo-app (CPA) is fairly minimal, yet is a good starting
+off point when creating simple or complex Prodo apps. It includes two [Prodo
+plugins](/basics/plugins): logger and route, which we will look at in the next section.
+
+CPA uses [Webpack](https://webpack.js.org/) to bundle your app and is setup with
+[hot module replacement
+(HMR)](https://webpack.js.org/concepts/hot-module-replacement/). This means when
+you make a change, your app will update without the page refreshing.
+
+Lets change the header in `src/components/Header.tsx`.
```
Comment (andrejak) [inline docs/content/05_tutorials/github-prs.md]:
```suggestion
Let's change the header in `src/components/Header.tsx`.
```

File: docs/content/05_tutorials/github-prs.md [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+```shell
+yarn start
+```
+
+Open up [localhost:8080](http://localhost:8080) and you should see this:
+
+![Prodo template first launch](prodo-template.png)
+
+The app created with create-prodo-app (CPA) is fairly minimal, yet is a good starting
+off point when creating simple or complex Prodo apps. It includes two [Prodo
```
Comment (andrejak) [inline docs/content/05_tutorials/github-prs.md]: I'd probably rephrase as "...fairly minimal, yet a good starting point..."

File: docs/content/05_tutorials/github-prs.md [hunk 3]
```diff
@@ -0,0 +1,493 @@
...
+
+# Specifying the model
+
+In a Prodo app your state is stored in a global store which is accessed by your
+components and updated by your actions. The type of this state is defined in a
+[model](/basics/model) in `src/model.ts`. Plugins can also be added to the model to extend the
+functionality.
+
+The model is created with `createModel` using the `State` type. Initially the
+app has not state. We also add the [logger](/plugins/logger) and
```
Comment (andrejak) [inline docs/content/05_tutorials/github-prs.md]:
```suggestion
app has no state. We also add the [logger](/plugins/logger) and
```

File: docs/content/05_tutorials/github-prs.md [hunk 4]
```diff
@@ -0,0 +1,493 @@
...
+```tsx
+import * as React from "react";
+import { Link } from "@prodo/route";
+
+export interface Props {
+  owner: string;
+  repo: string;
+}
+
+const Repo: React.FC<Props> = ({ owner, repo }) => (
```
Comment (andrejak) [inline docs/content/05_tutorials/github-prs.md]: Is it worth explaining React.FC?

Comment (coffee-cup) [reply docs/content/05_tutorials/github-prs.md]: I don't think so, we are assuming they know React

**Comment by andrejak** [comment]
Also I think the best way to test the usefulness of the tutorial is to have someone external go through it, so getting this merged soon & editing after seems sensible.

**Pull request merged**